update-sri.js

```diff
@@ -35,7 +35,7 @@ Promise.all(process.argv.slice(3).map(file =>
             return pre + version + post;
         });
         return Promise.all(Object.keys(hashes).map(hash =>
-            read(path.join("dist", hashes[hash].file), null)
+            read(hashes[hash].file, null)
```

Comment:
why did this change?

Reply:
jsDelivr links already include `dist`, see above in readme

Reply:
Cool.  I double checked the regex on regex101.com and it checks out.  The regex could probably be simplified, but your changes are minimal so it's okay.